app/src/main/java/edu/usf/sas/pal/muser/util/EventUtils.java

```diff
@@ -0,0 +1,43 @@
+package edu.usf.sas.pal.muser.util;
+
+import android.content.Context;
+
+import com.simplecity.amp_library.ShuttleApplication;
+import com.simplecity.amp_library.model.Song;
+
+import edu.usf.sas.pal.muser.firebase.model.Event;
+import edu.usf.sas.pal.muser.firebase.model.SongData;
+
+
+
+public class EventUtils {
+
+    /**
+     * Function to populate the Event data class.
+     * @param song - The song for which the event occurred.
+     * @param capturedEvent - The event that was captured.
+     * @param context - The context of the Fragment.
+     * @return - Event object
+     */
+    public static Event newEvent(Song song, String capturedEvent, Context context){
```

Comment:
We don't need `context` here, right?

Reply:
Oh, nevermind, I just saw your other comment 👍

Reply:
I am passing the context received here into the `newSongData` function.